api/api.pb.go

```diff
@@ -1062,53 +1062,53 @@ func (m *Group) GetId() string {
 // swagger:model
 type VolumeSpec struct {
 	// Ephemeral storage
-	Ephemeral bool `protobuf:"varint,1,opt,name=ephemeral,proto3" json:"ephemeral,omitempty"`
```

Comment:
Why are all the proto3 references deleted? did we change versions? does it not matter?

Reply:
Great question. It all depends who generated them before and after. Right before we 'release', we should agree which protoc version to use and generate using that.